Fixes bugs in the state tomography measurement:

stephlazar opened the pull request:
Fixes the problems:
 - the sweep points and detector function were not configured correctly; measurement never finishes
 - the multiplexed_pulse function was not called correctly in the state tomography sequence functions

Commit:
Fixes bugs in the state tomography measurement:
 - the measurement was not configured correctly
 - the multiplexed_pulse function was not called correctly

antsr commented on pycqed/measurement/multi_qubit_module.py:
```diff
@@ -934,14 +921,14 @@ def measure_tomography(qubits, prep_sequence, state_name,
         MC.run_2D(label, exp_metadata=exp_metadata)
 
-    return 'Tomo'
+    return
```
useless return statement

stephlazar replied:
I agree. Will remove.

antsr commented on pycqed/measurement/pulse_sequences/multi_qubit_tek_seq_elts.py:
```diff
@@ -980,10 +981,8 @@ def n_qubit_tomo_seq(
-                                                          'RO_presel',
-                                                          True, -ro_spacing)
+            ro_pulses_presel = generate_mux_ro_pulse_list(
+                qubit_names, operation_dict, 'RO_presel', 'end', -ro_spacing)
```
I am suprised that the preselection code was buggy for so long (I checked that the fourth argument to the function should not have been a boolean even in August), good to have it fixed.
Maybe `True` still worked somehow.

But wouldn't it be more intuitive if `ro_spacing` was the timing between the starts  of the two readouts?
For that the third argument would have to be changed to `'start'`.

stephlazar replied:
Should I also change the default value of this input parameter in the generate_mux_ro_pulse_list function? This might break some measurements like the multi_parity_multi_round_seq or others that are not even in the master.

antsr replied:
I think the default should stay as it is, because in the typical case where we want to read out the qubits at the end of the experiment, the `ref_point='end'` makes most sense.
Only for preselection we want to set `ref_point='start'` and `pulse_delay=-ro_spacing`

stephlazar replied:
I see. Ok

antsr replied:
I found the commit from June that changed the `first_pulse_start=False` argument to `ref_point='end'` argument.
So `first_pulse_start=True` should translate to `ref_point='start'`

stephlazar replied:
One gets an error though when one tries to run the measurement. Because the function expects a string instead of a bool?

antsr replied:
could be because of that, yes.
I am not sure what `Segment` does when the ref-point is set to a bool

stephlazar replied:
Probably fails:

![image](https://user-images.githubusercontent.com/14844492/77530112-251a1100-6e91-11ea-8682-405884cbc652.png)

antsr commented on pycqed/measurement/pulse_sequences/multi_qubit_tek_seq_elts.py:
```diff
@@ -1070,10 +1068,8 @@ def n_qubit_ref_seq(qubit_names, operation_dict, ref_desc, upload=True,
-                                                          'RO_presel',
-                                                          True, -ro_spacing)
+            ro_pulses_presel = generate_mux_ro_pulse_list(
+                qubit_names, operation_dict, 'RO_presel', 'end', -ro_spacing)
```
same as above `'end'` -> `'start'`

Commit:
Removes redundant return statement in measure_tomography.
Replaces the refpoint 'end' with 'start' in call to generate_mux_ro_pulse_list in n_qubit_tomo_seq and n_qubit_ref_seq.

nathlacroix reviewed (commented):
Looks good to me.
At some point will need another cleanup round. e.g. to remove all the "return_seq" arguments. IMO sequence should always be returned, also the way we add calibration points is a bit weird, I would generate everything from the same sequence. But otherwise it's all good 👍

Commit:
Merge branch 'qudev_master' into bugfix/state_tomography
